src/clib/pioc_support.c

```diff
@@ -170,6 +170,7 @@ void pio_log(int severity, const char *fmt, ...)
 {
     va_list argp;
     int t;
+    int rem_len = MAX_LOG_MSG;
     char msg[MAX_LOG_MSG];
```

Comment:
You want msg to be MAX_LOG_MSG + 1.

The plus one is for the null that is on the end.